command/settings/update.go

```diff
@@ -0,0 +1,208 @@
+
+		&cli.StringFlag{
+			EnvVars: []string{"VELA_COMPILER_CLONE_IMAGE", "COMPILER_CLONE_IMAGE"},
+			Name:    CompilerCloneImageKey,
+			Aliases: []string{"clone", "clone-image", "cloneimage", "ci"},
+			Usage:   "image used to clone the repository for the compiler",
+		},
+
+		&cli.IntFlag{
+			EnvVars: []string{"VELA_COMPILER_STARLARK_EXEC_LIMIT", "COMPILER_STARLARK_EXEC_LIMIT"},
+			Name:    CompilerStarlarkExecLimitKey,
+			Aliases: []string{"starlark-exec-limit", "starklark-limit", "starlarklimit", "sel"},
+			Usage:   "max starlark execution limit for the compiler",
+		},
+
+		// Misc Flags
+
+		&cli.StringSliceFlag{
+			EnvVars: []string{"VELA_REPO_ALLOWLIST_ADD_REPOS", "REPO_ALLOWLIST_ADD_REPOS"},
+			Name:    RepoAllowlistAddKey,
+			Aliases: []string{"repo", "repos", "ral"},
+			Usage:   "the list of repositories to add to the list of all those permitted to use Vela",
+		},
+
+		&cli.StringSliceFlag{
+			EnvVars: []string{"VELA_REPO_ALLOWLIST_DROP_REPOS", "REPO_ALLOWLIST_DROP_REPOS"},
+			Name:    RepoAllowlistDropKey,
+			Usage:   "the list of repositories to drop from the list of all those permitted to use Vela",
+		},
+
+		&cli.StringSliceFlag{
+			EnvVars: []string{"VELA_SCHEDULE_ALLOWLIST_ADD_REPOS", "SCHEDULE_ALLOWLIST_ADD_REPOS"},
+			Name:    ScheduleAllowAddlistKey,
+			Aliases: []string{"schedule", "schedules", "sal"},
+			Usage:   "the list of repositories to add to the list of all those permitted to use schedules in Vela",
+		},
+
+		&cli.StringSliceFlag{
+			EnvVars: []string{"VELA_SCHEDULE_ALLOWLIST_DROP_REPOS", "SCHEDULE_ALLOWLIST_DROP_REPOS"},
+			Name:    ScheduleAllowDroplistKey,
+			Usage:   "the list of repositories to drop from the list of all those permitted to use schedules in Vela",
+		},
+
+		&cli.StringFlag{
+			EnvVars: []string{"VELA_FILE", "SECRET_FILE"},
+			Name:    "file",
+			Aliases: []string{"f"},
+			Usage:   "provide a file to update settings",
+		},
+
+		// Output Flags
+
+		&cli.StringFlag{
+			EnvVars: []string{"VELA_OUTPUT", "SETTINGS_OUTPUT"},
+			Name:    internal.FlagOutput,
+			Aliases: []string{"op"},
+			Usage:   "format the output in json, spew or yaml",
+			Value:   "yaml",
+		},
+	},
+	CustomHelpTemplate: fmt.Sprintf(`%s
+EXAMPLES:
+  1. Update settings to change the compiler clone image to target/vela-git:latest.
+    $ {{.HelpName}} --compiler.clone-image target/vela-git:latest
+  2. Update settings to change the compiler template depth to 2.
+    $ {{.HelpName}} --compiler.template-depth 2
+  3. Update settings to change the compiler starlark exec limit to 5.
+    $ {{.HelpName}} --compiler.starlark-exec-limit 5
+  4. Update settings with additional queue routes.
+    $ {{.HelpName}} --queue.add-route large --queue.add-route small
+  5. Update settings by dropping queue routes.
+    $ {{.HelpName}} --queue.drop-route large --queue.drop-route small
+  6. Update settings with additional repos permitted to use Vela.
+    $ {{.HelpName}} --add-repo octocat/hello-world --repo octocat/*
```

Comment:
```suggestion
    $ {{.HelpName}} --add-repo octocat/hello-world --add-repo octocat/*
```

I get that this would technically work because of the alias, but to me this is a little confusing

Reply:
oh sure i put it there like that on purpose but i agree, looks like a typo